bundles/framework/maplegend/Flyout.js

```diff
@@ -52,9 +59,7 @@ Oskari.clazz.define('Oskari.mapframework.bundle.maplegend.Flyout',
          */
         startPlugin: function () {
             var me = this;
-            me.templateLayer = jQuery('<div class="maplegend-layer"></div>');
             me.templateTools = jQuery('<div class="maplegend-tools"><div class="layer-description"><div class="icon-info"></div></div></div>');
```

Comment:
Can this be removed as well?

Reply:
Thought it would break something. Apparently no. Deleted it.

Reply:
It's still in the code. GitHub will resolve the conversation automatically when the line is removed.